srcpkgs/pysolfc/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'pysolfc'
 pkgname=pysolfc
 version=2.6.4
-revision=1
+revision=2
 arch="noarch"
```

Comment:
please replace with `archs=noarch`, this is an actual error in the template and should be fixed.

Reply:
Done